### PR TITLE
docs: add github processed templates

### PR DESCRIPTION
**Pull request by sergiusens**

### CODE REVIEW

**Comment by kalikiana** [comment]
Are these magic files to be used by GitHub? I can't find any documentation on those.

**Comment by sergiusens** [comment]
@elopio why remove `PULL_REQUEST_TEMPLATE.md`? This adds the checklist on the pr list view!!

@kalikiana I found them through other projects.

**Comment by elopio** [comment]
@sergiusens you now have two PULL_REQUEST_TEMPLATE.md files.
You should remove the old one, so we use the one in the .github directory: 
https://github.com/snapcore/snapcraft/blob/master/PULL_REQUEST_TEMPLATE.md

**Comment by sergiusens** [comment]
@elopio ah, thanks for making that clearer :-)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- [ ] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
+- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
+- [ ] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
+- [ ] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
+- [ ] Have you succesfully run `./runtests.sh static`?
```
Comment (sparkiegeek) [inline .github/PULL_REQUEST_TEMPLATE.md]: typo on successfully

**Pull request merged**